documentation/biblatex-manuscripts-philology.tex

```diff
@@ -65,26 +71,41 @@ \subsection{Mandatory}
 
 \begin{fieldlist}
 
-\fielditem{collection}{literal} the collection in the library. For example: \verb+Supplément grec+.
+\fielditem{collection}{literal} the collection in the library. For example: 
+\verb+Supplément grec+ or \verb+Vaidyaka+.
 
-\fielditem{location}{literal} the city or place where the manuscript is kept. For example: \verb+Paris+ or \verb+Oxford+.
+\fielditem{location}{literal} the city or place where the manuscript is kept. For 
+example: \verb+Paris+ or \verb+Oxford+ or \verb+Varanasi+.
 
-\fielditem{library}{literal} the library where the manuscript is kept. For example: \verb+Bibliothèque Nationale de France+.
+\fielditem{library}{literal} the library where the manuscript is kept. For example: 
+\verb+Bibliothèque Nationale de France+ or \verb+Sarasvatī Bhavan Library+.
 
 
-\fielditem{shelfmark}{literal} the shelfmark in the collection. For example: \verb+241+.
+\fielditem{shelfmark}{literal} the shelfmark in the collection. For example: 
+\verb+241+.  Do not prefix with “MS.”
 \end{fieldlist}
 
 \subsection{Optional}
 This bibtype can use the optional fields:
 
 \begin{fieldlist}
 
-\fielditem{bookpagination}{key} the pagination of the manuscript which is studied. The standard pagination keys are allowed, but the package adds a new key: \texttt{folio}. The value of this field is to be used for printing the \bibfield{pages} and \bibfield{pagetotal}.
+\fielditem{bookpagination}{key} the pagination or foliation of the manuscript 
+which is studied. The standard \biblatex pagination keys are allowed, but the 
+package adds a new key: \texttt{folio}. 
+
+The value of this field will be used
+when printing the \bibfield{pages} and \bibfield{pagetotal}.
+E.g., ``ff.\ 3--5'' (when  \bibfield{bookpagination} =
+“folio”) or ``pp.\ 88--99'' (“page”).  See the \biblatex manual for details.
 
 \fielditem{columns}{integer} the number of column by pages. Basically, only two numbers are allowed: \verb+1+ or \verb+2+.
 
-\fielditem{dating}{literal} the dating of the manuscript. It can be for example a century.
+\fielditem{dating}{literal} the dating of the manuscript. It can be for example a 
+century.  
+
+With non-Gregorian calendars, enter the converted Gregorian date here, and the 
+date in the original era in the \bibfield{annotation} field.
```

Comment:
Is it the best way? As the`dating` field is is literal, we may add the two forms in the same place. In another way, we could create a specific field. Annotation field is normally for long commentary. 

Alos, we could look on biblatex recent change in order to see if they can manage two parallel calendar system.

Reply:
Yes.  I was wrong.  One should give the date as stated in the manuscript, followed by the Gregorian date in square brackets.  See the conventions of the Anglo-American Cataloguing Rules 2, section 1.4F1 (https://archive.org/stream/pdfy-VV-4_m1HVoyDFFrA/AACR2#page/n57).
I haven't looked at the latest biblatex documentation today, but in fact having two fields - one literal and one controlled - is quite a good idea, to help with sorting.

Reply:
for brackets, you can use \mkbibrackets, which will take account of nested brackets.